### PR TITLE
chore: connect index check linter to intellij INTELLIJ-72

### PR DESCRIPTION
**Pull request by kmruiz**
### Checklist

- [ ] New tests and/or benchmarks are included.
- [ ] Documentation is changed or added.
- [ ] Changelog is updated accordingly.
- [ ] I have signed the MongoDB Contributor License Agreement (https://www.mongodb.com/legal/contributor-agreement).

### CODE REVIEW

File: packages/mongodb-access-adapter/datagrip-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/datagrip/adapter/DataGripMongoDbDriver.kt [hunk 1]
```diff
@@ -79,6 +81,8 @@ internal class DataGripMongoDbDriver(
 
     override suspend fun connectionString(): ConnectionString = ConnectionString(dataSource.url!!)
 
+    override suspend fun <S> explain(query: Node<S>): ExplainPlan = ExplainPlan.CollectionScan
```
Comment (himanshusinghs) [inline packages/mongodb-access-adapter/datagrip-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/datagrip/adapter/DataGripMongoDbDriver.kt:84]: I guess this is the hardcoded part?

File: packages/mongodb-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/slice/ExplainQuery.kt [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ * @property explainPlan
+ */
+data class ExplainQuery(
+    val explainPlan: ExplainPlan
+) {
+/**
+ * @param S
+ * @property query
+ */
+data class Slice<S>(val query: Node<S>) : com.mongodb.jbplugin.accessadapter.Slice<ExplainQuery> {
```
Comment (himanshusinghs) [inline packages/mongodb-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/slice/ExplainQuery.kt:23]: nit: I think formatting got missed here

**Review by himanshusinghs — APPROVED** [review]
Great work 🚀

File: packages/mongodb-access-adapter/datagrip-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/datagrip/adapter/DataGripMongoDbDriver.kt (hunk 1 above)
Comment (kmruiz) [reply packages/mongodb-access-adapter/datagrip-access-adapter/src/main/kotlin/com/mongodb/jbplugin/accessadapter/datagrip/adapter/DataGripMongoDbDriver.kt:84]: Yes! We are not running the explain yet so I'm just returning CollectionScan in the meantime until we have a proper implementation.

**Pull request merged**